weepickle/src/main/scala/com/rallyhealth/weepickle/v1/JsFromTos.scala

```diff
@@ -35,4 +35,58 @@ trait FromToValue extends MacroImplicits { this: Types with Annotator =>
     new From[com.rallyhealth.weejson.v1.Value] {
       def transform0[Out](v: Value, out: Visitor[_, Out]): Out = v.transform(out)
     }
+
+  /*
+   * And the same for BufferedValue types
+   */
+  implicit val ToBufferedValue: To[com.rallyhealth.weejson.v1.BufferedValue] =
+    new To.Delegate(com.rallyhealth.weejson.v1.BufferedValue.Builder)
+
+  implicit def ToBufferedValueObj: To[com.rallyhealth.weejson.v1.BufferedValue.Obj] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Obj]
+  implicit def ToBufferedValueArr: To[com.rallyhealth.weejson.v1.BufferedValue.Arr] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Arr]
+  implicit def ToBufferedValueStr: To[com.rallyhealth.weejson.v1.BufferedValue.Str] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Str]
+  implicit def ToBufferedValueAnyNum: To[com.rallyhealth.weejson.v1.BufferedValue.AnyNum] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.AnyNum]
+  implicit def ToBufferedValueNum: To[com.rallyhealth.weejson.v1.BufferedValue.Num] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Num]
+  implicit def ToBufferedValueNumLong: To[com.rallyhealth.weejson.v1.BufferedValue.NumLong] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.NumLong]
+  implicit def ToBufferedValueNumDouble: To[com.rallyhealth.weejson.v1.BufferedValue.NumDouble] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.NumDouble]
+  implicit def ToBufferedValueBinary: To[com.rallyhealth.weejson.v1.BufferedValue.Binary] =
+    ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Binary]
+  implicit def ToBufferedValueExt: To[com.rallyhealth.weejson.v1.BufferedValue.Ext] =
+    ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Ext]
+  implicit def ToBufferedValueTimestamp: To[com.rallyhealth.weejson.v1.BufferedValue.Timestamp] =
+    ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Timestamp]
+  implicit def ToBufferedValueBool: To[com.rallyhealth.weejson.v1.BufferedValue.Bool] = ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Bool]
+  implicit def ToBufferedValueTrue: To[com.rallyhealth.weejson.v1.BufferedValue.True.type] =
+    ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.True.type]
+  implicit def ToBufferedValueFalse: To[com.rallyhealth.weejson.v1.BufferedValue.False.type] =
+    ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.False.type]
+  implicit def ToBufferedValueNull: To[com.rallyhealth.weejson.v1.BufferedValue.Null.type] =
+    ToBufferedValue.narrow[com.rallyhealth.weejson.v1.BufferedValue.Null.type]
```

Comment:
Haven't tested, but would this work?

```suggestion
private val ToValue: To[com.rallyhealth.weejson.v1.Value] =
  new To.Delegate(com.rallyhealth.weejson.v1.Value)

implicit def ToValueObj[T <: com.rallyhealth.weejson.v1.Value]: To[T] = ToValue.narrow[T]
```

Reply:
Oh, it trips up `compileError("implicitly[com.rallyhealth.weepickle.v1.WeePickle.To[Nothing]]")`. Nevermind!